### PR TITLE
Feature/updated dark

### PR DESCRIPTION
**Pull request by boufni95**

### CODE REVIEW

**Review by danlugo92 — CHANGES_REQUESTED** [review]

**Comment by danlugo92** [comment]
Bottom bar colors are inverted,

Arturo: White when selected, blue otherwise

Code: Blue when selected, white otherwise

**Comment by danlugo92** [comment]
Advanced screen: 

Transactions, channels, peers etc have black text on a dark gray background, non-legible

File: android/settings.gradle [hunk 1]
```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'shockwallet'
+include ':react-native-svg'
+project(':react-native-svg').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-svg/android')
```
[the remark this answers is not in the document]
Comment (boufni95) [reply android/settings.gradle]: removed

File: android/app/src/main/java/com/shockwallet/MainApplication.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import android.content.Context;
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.horcrux.svg.SvgPackage;
```
[the remark this answers is not in the document]
Comment (boufni95) [reply android/app/src/main/java/com/shockwallet/MainApplication.java]: removed

File: android/app/build.gradle [hunk 3]
```diff
@@ -202,6 +202,7 @@ configurations.all {
 }
 
 dependencies {
+    implementation project(':react-native-svg')
```
[the remark this answers is not in the document]
Comment (boufni95) [reply android/app/build.gradle]: removed

File: ios/Podfile [hunk 4]
```diff
@@ -34,6 +34,8 @@ target 'shockwallet' do
 
   pod 'react-native-webview', :path => '../node_modules/react-native-webview'
 
+  pod 'RNSVG', :path => '../node_modules/react-native-svg'
```
[the remark this answers is not in the document]
Comment (boufni95) [reply ios/Podfile]: removed

File: app/screens/MyFeed/index.tsx [hunk 5]
```diff
@@ -0,0 +1,521 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (boufni95) [reply app/screens/MyFeed/index.tsx]: removed

File: app/screens/MyFeed/SetBioDialog.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/jsx-no-bind */
```
[the remark this answers is not in the document]
Comment (boufni95) [reply app/screens/MyFeed/SetBioDialog.js]: removed

File: app/screens/MyFeed/MetaConfigModal.tsx [hunk 7]
```diff
@@ -0,0 +1,165 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (boufni95) [reply app/screens/MyFeed/MetaConfigModal.tsx]: removed

File: app/res/css/index.js [hunk 8]
```diff
@@ -44,7 +45,7 @@ export const styles = StyleSheet.create({
   fontSize22: { fontSize: 22 },
   fontSize24: { fontSize: 24 },
   height0: { height: '0%' },
-  height100: { height: '100%' },
+  height100: { height: '100%', backgroundColor: '#16191C' },
```
[the remark this answers is not in the document]
Comment (boufni95) [reply app/res/css/index.js]: removed

**Comment by danlugo92** [comment]
https://imgur.com/a/3VBa4X7

Few photos showing discrepancy between current dark mode code and Arturo designs

**Pull request merged**